installer/app/src/views/azure-create-app-tutorial.js.jsx

```diff
@@ -5,163 +5,240 @@ import Sheet from './css/sheet';
+                    }} />
+                <p>Click the "Add" button at the bottom and select "Add an application my organization is developing". For the name, choose "flynn-installer" (or something else), then select "Native Client Application".</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-04">
+            <img src={AssetPaths["azure-4.gif"]} style={{
+                        width: 699,
+                        height: 378
+                    }} />
+                <p>As Redirect URI, use the value below, and create the application by hitting the checkmark in the lower right.</p>
+            </div>
+        );
+
+
+        tutorialSlides.push(
+            <div id="tutorial-05">
+            <img src={AssetPaths["azure-5.gif"]} style={{
+                        width: 699,
+                        height: 378
+                    }} />
+                <p>Click the "Configure" tab and copy the "Client ID" into the input field below.</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-06">
+                <img src={AssetPaths["azure-6.gif"]} style={{
+                        width: 699,
+                        height: 402
+                    }} />
+                <p>Next, we need to allow the created app to control your Azure account. Scroll to the bottom of the configuration page and click the green "Add application" button. In the popup, select the "Windows Azure Service Management API" and click the checkmark in the lower right.</p>
+            </div>
+        );
+
+
+        tutorialSlides.push(
+            <div id="tutorial-07">
+                <img src={AssetPaths["azure-7.gif"]} style={{
+                        width: 699,
+                        height: 402
+                    }} />
+                <p>Click the "Delegated Permissions" dropdown and check "Access Azure Service Management". Then, save the configuration.</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-08">
+                <img src={AssetPaths["azure-8.gif"]} style={{
+                        width: 699,
+                        height: 472
+                    }} />
+                <p>Click on the back arrow button to go back to the "APPLICATIONS" tab and click and the "ENDPOINTS" button at the bottom. Then, copy your OAuth 2.0 Token Endpoint into the input below.</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-09">
+                <img src={AssetPaths["azure-done.png"]} style={{
+                        width: 699,
+                        height: 256
+                    }} />
+                <p>You now created an appliation able to control your Azure resources - one that this Flynn installer can use. You can connect this installer and your new app by clicking "Authenticate" below - lets install Flynn!</p>
+            </div>
+        );
```

Comment:
`tutorialSlides` can be defined above the React class and the divs don't need ids.